yb-voyager/cmd/root.go

```diff
@@ -46,7 +46,7 @@ Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		if exportDir != "" && utils.FileOrFolderExists(exportDir) {
 			// TODO: avoid lock only for fall-forward setup
-			if cmd.Use != "version" && cmd.Use != "status" && cmd.Use != "initiate" && !(cmd.Use == "data" && cmd.Parent().Use == "import"){
+			if cmd.Use != "version" && cmd.Use != "status" && cmd.Use != "initiate" && !(cmd.Use == "setup" && cmd.Parent().Use == "fall-forward") {
```

Comment:
nit: maybe a separate function and array to store these commands names can be introduced here.

Reply:
this condition can also be modified: `cmd.Use != "initiate"`
to
`!(cmd.Use != "initiate" && cmd.Use!="cutover")

I skipped it since cutover is only command command having `initiate` 
But lets do it for both then